exchanges/stream/websocket.go

```diff
@@ -24,8 +24,10 @@ const jobBuffer = 5000
 var (
 	ErrWebsocketNotEnabled      = errors.New("websocket not enabled")
 	ErrSubscriptionFailure      = errors.New("subscription failure")
-	ErrSubscriptionNotSupported = errors.New("subscription channel not supported ")
 	ErrUnsubscribeFailure       = errors.New("unsubscribe failure")
+	ErrSubscriptionNotSupported = errors.New("subscription channel not supported ")
```

Comment:
Whilst noticing the space after this, also noticed that this is unused so can be removed

Reply:
[thanks](https://github.com/thrasher-corp/gocryptotrader/pull/1717/commits/68461c546d5d3ec4fc82e338338091bfa1d66c71)